enterprise_access/apps/subsidy_request/models.py

```diff
@@ -306,3 +307,86 @@ def update_course_info_for_subsidy_request(sender, **kwargs):  # pylint: disable
         subsidy_type,
         str(subsidy_request.uuid),
     )
+
+
+class LearnerCreditRequest(SubsidyRequest):
```

Comment:
I think we should have an additional field to save `course_key`. I don't think the `SubsidyRequest` model currently stores it. It only has `course_run_id`.

Reply:
`SubsidyRequest` has the `course_id` field.
https://github.com/openedx/enterprise-access/blob/de56f3b4d8ecc8188bb4b5671e5ca75f78cc11df/enterprise_access/apps/subsidy_request/models.py#L47